<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialogClass</class>
 <widget class="QDialog" name="CSettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Qrfe/images/qrfe.qrc">
    <normaloff>:/ams logos/mtiWindowIcon</normaloff>:/ams logos/mtiWindowIcon</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../Qrfe/images/qrfe.qrc">:/window images/settingsImage</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Display Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="useTtlCheckBox">
        <property name="text">
         <string>Use Time To Live</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QLabel" name="inactiveMSLabel">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>msecs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="inactiveSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="inactiveLabel">
          <property name="text">
           <string>Show tag inactive after</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="outOfRangeMSLabel">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>msecs</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="outOfRangeSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="outOfRangeLabel">
          <property name="text">
           <string>Show tag out of range after</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="deleteMSLabel">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>msecs</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="deleteSpinBox">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="deleteLabel">
          <property name="text">
           <string>Delete tag after</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="showAliasCheckBox">
        <property name="text">
         <string>Show alias name if exists</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Scan Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="useMultiplexBox">
        <property name="text">
         <string>Multiplex Reader</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="multiplexReadTimeLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Read Time:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="multiplexReadTimeBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="multiplexTimeLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>msecs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Trace</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="useTraceBox">
        <property name="text">
         <string>Use Trace</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="traceLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Trace:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="traceLevelBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="traceLevelLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Level</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Register Map Update</string>
     </property>
     <widget class="QCheckBox" name="registerMapUpdateEnable">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>24</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="pollingText">
         <property name="text">
          <string>Polling Interval :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pollingIntervalSB">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pollingTimeLabel">
         <property name="text">
          <string>msecs</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
  <include location="../../Qrfe/images/qrfe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>inactiveSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outOfRangeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>deleteSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>inactiveLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>inactiveMSLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outOfRangeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outOfRangeMSLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>deleteLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTtlCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>deleteMSLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTraceBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>traceLevelBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTraceBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>traceLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useTraceBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>traceLevelLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useMultiplexBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>multiplexReadTimeBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useMultiplexBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>multiplexReadTimeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useMultiplexBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>multiplexTimeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registerMapUpdateEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pollingIntervalSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registerMapUpdateEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pollingText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>registerMapUpdateEnable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pollingTimeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
